internal/dlc/dlc.go

```diff
@@ -114,3 +119,22 @@ func (b *Builder) CopyReqsFromCounterparty(d *DLC) {
 	b.dlc.fundTxReqs.txIns[p] = fundReqs.txIns[p]
 	b.dlc.fundTxReqs.txOut[p] = fundReqs.txOut[p]
 }
+
+// AcceptCounterpartySign verifies couterparty's given sign is valid and then
+func (b *Builder) AcceptCounterpartySign(sign []byte) error {
+	p := b.dlc.counterparty(b.party)
+
+	err := b.dlc.VerifyRefundTx(sign, b.dlc.pubs[p])
+	if err != nil {
+		return fmt.Errorf("counterparty's signature didn't pass verification, had error: %v", err)
+	}
+
+	// sign passed verification, accept it
+	b.dlc.refundSigns[p] = sign
+	return nil
+}
+
+//SetLockTime sets the locktime of DLC
+func (b *Builder) SetLockTime(locktime uint32) {
+	b.dlc.lockTime = locktime
```

Comment:
The locktime should be set when initializing DLC as it's a condition of the contract.

Reply:
I think in another PR `DLC` should be initialized with `locktime, fundFeerate, redeemFeerate`, but for now I think its ok that `DLC` can be initialized without those values. So I think, for now, its ok to keep this setting function